packages/release-manifests/api-report.md

```diff
@@ -0,0 +1,34 @@
+## API Report File for "@backstage/release-manifests"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+// @public
```

Comment:
should add this package to `NO_WARNING_PACKAGES` https://github.com/backstage/backstage/blob/9370194ce9e52b8abeeda06539b67bb34b323c54/scripts/api-extractor.ts#L201